Makefile

```diff
@@ -1,8 +1,8 @@
 compile: src/search_client.erl
-	@rebar3 compile
+	@rebar compile
```

Comment:
This needs to change back to `rebar3`. I was trying different things for compiling inside a couchdb build, and this didn't work out. This won't matter because we will have a separate branch just to generate code using `rebar3` and the Makefile in this branch will never be called.